doc/dev/devices.rst

```diff
@@ -0,0 +1,87 @@
+
+.. raw:: html
+
+    <style>
+    tr {
+        border-top: #d3d3d3 1px solid;
+        border-bottom: #d3d3d3 1px solid;
+    }
+    .row-odd {
+        background-color: #f7f7f7;
+    }
+    </style>
+
+Supported devices
+=================
+
+Not all PennyLane devices currently work with Catalyst, and for those that do, their supported
+feature set may not necessarily match supported features when used without :func:`~.qjit`.
+
+Supported backend devices include:
+
+.. list-table::
+  :widths: 20 80
+
+  * - ``lightning.qubit``
+
+    - A fast state-vector qubit simulator written with a C++ backend. See the
+      `Lightning documentation <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_qubit/device.html>`__
+      for more details, as well as its
+      `Catalyst configuration file <https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/lightning_qubit/lightning_qubit.toml>`__.
+
+  * - ``lightning.kokkos``
+
+    - A fast state-vector qubit simulator utilizing the Kokkos library for CPU and GPU accelerated
+      circuit simulation. See the
+      `Lightning documentation <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_kokkos/device.html>`__
+      for more details, as well as its
+      `Catalyst configuration file <https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/lightning_kokkos/lightning_kokkos.toml>`__.
```

Comment:
@maliasadi is there any documentation I can link to for installing kokkos + catalyst?

Reply:
pip installing pennylane-lightning-kokkos, just like for pennylane-lightning, is sufficient to get it running. However, it would only include the CPU implementation as far as know, for GPU from source installation by the user would be required.